components/datetime/src/provider/calendar/mod.rs

```diff
@@ -96,6 +102,7 @@ pub mod patterns {
         pub short: runtime::Pattern<'data>,
     }
 
+    /// Data struct for date/time patterns based on plural category, broken down by pattern length.
```

Comment:
Observation: is `LengthPatternPluralsV1` actually being used anywhere? I don't see it in any searches. Please delete it if possible.